tests/functional/RemoteTargetLocatorTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+// Copyright 2019-present Facebook. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Facebook\WebDriver\Remote;
+
+use Facebook\WebDriver\WebDriverTestCase;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\Exception\InvalidArgumentException;
+
+/**
+ * @covers \Facebook\WebDriver\Remote\RemoteTargetLocator
+ */
+class RemoteTargetLocatorTest extends WebDriverTestCase
+{
+    /**
+     * @cover ::activeElement
+     */
+    public function testActiveElement()
+    {
+        $this->driver->get($this->getTestPageUrl('index.html'));
+
+        $activeElement = $this->driver->switchTo()->activeElement();
+        $this->assertInstanceOf(\Facebook\WebDriver\Remote\RemoteWebElement::class, $activeElement);
```

Comment:
You can import the class and have there only:

```suggestion
        $this->assertInstanceOf(RemoteWebElement::class, $activeElement);
```